programs/survival/bucketstack.sc

```diff
@@ -0,0 +1,223 @@
+__config() ->
+(
+	m(
+		l('scope','player'),
+		l('stay_loaded',true),
+
+		l('commands',
+			m(
+				l('<bucket>','__print'),
+				l('<bucket> <stack>','__change')
+			)

```

Comment:
how about adding an `/bucketstack all <size>` for a global stack size for all bucket types?

Reply:
Seems fine to me.